tests/elasticsearch/basic.bats

```diff
@@ -64,9 +58,7 @@ DETIK_CLIENT_NAMESPACE="${TEST_NAMESPACE}"
         --timeout "${HELM_FB_TIMEOUT:-5m0s}" \
         --wait
 
-    try "at most 15 times every 2s " \
-        "to find 1 pods named 'fluent-bit' " \
-        "with 'status' being 'running'"
+    kubectl wait pods -n "$TEST_NAMESPACE" -l app.kubernetes.io/name=fluent-bit --for condition=Ready --timeout=30s
```

Comment:
This has a race I think, it will fail if when it checks the first time there is nothing matching the selector. It's a frustration with wait that it waits or fails, not waits if it fails.

We can define the default namespace for Detik to use so why change it?

Reply:
ahh! great catch, let me change these back!